
fx: 500
fy: 500
cx: 400
cy: 300
image_width: 640
image_height: 480
horizontal_views: 10
vertical_views: 8
distance: 2.7
batch_size: 50
